Dockerfile

```diff
@@ -1,13 +1,14 @@
 FROM mhart/alpine-node:0.10.48
 MAINTAINER butlerx <butlerx@notthe.cloud>
+ENV NODE_ENV=production
 ARG DEP_VERSION=latest
-RUN apk add --update git build-base python postgresql-client
-RUN mkdir -p /usr/src/app
+RUN apk add --update git build-base python postgresql-client &&\
+   mkdir -p /usr/src/app
 WORKDIR /usr/src/app
-ADD . /usr/src/app
+COPY . /usr/src/app
 RUN npm install && \
-    npm install cp-translations@$DEP_VERSION && \
+    npm install cp-translations@"$DEP_VERSION" && \
```

Comment:
Ar eyou sure those double quotes won't be interepreted during the npm install ?

Reply:
nope its just shell script docker treats everything in a run command as sh